packages/atlas/src/views/viewer/DiscoverView/DiscoverView.tsx

```diff
@@ -57,20 +55,10 @@ export const DiscoverView: React.FC = () => {
     return null
   }, [categories, mappedVideoCategories, categoriesFeaturedVideos])
 
-  const { videoCount, error } = useVideoCount(
-    {},
-    {
-      onError: (error) => SentryLogger.error('Failed to fetch videos count', 'DiscoverView', error),
-    }
-  )
   const isMdBreakpoint = useMediaMatch('md')
 
   const headTags = useHeadTags('Discover')
 
-  if (error) {
-    return <ViewErrorFallback />
-  }
-
```

Comment:
It wouldn't hurt to keep this error fallback but instead, check if an error comes from the `useCategory` hook